### PR TITLE
slcli vlan list only returns 100 results

### PR DESCRIPTION
**Pull request by caberos**
[#issue1851](https://github.com/softlayer/softlayer-python/issues/1851)

image

### CODE REVIEW

File: SoftLayer/managers/network.py [hunk 1]
```diff
@@ -551,7 +551,7 @@ def list_vlans(self, datacenter=None, vlan_number=None, name=None, limit=100, **
 
         kwargs['iter'] = True
         if limit > 0:
-            return self.account.getNetworkVlans(mask=kwargs['mask'], filter=_filter.to_dict(), limit=limit)
+            return self.account.getNetworkVlans(mask=kwargs['mask'], filter=_filter.to_dict(), limit=limit, iter=False)
```
Comment (allmightyspiff) [inline SoftLayer/managers/network.py:554]:
```suggestion
            return self.account.getNetworkVlans(mask=kwargs['mask'], filter=_filter.to_dict(), limit=limit, iter=True)
```

**Review by allmightyspiff — CHANGES_REQUESTED** [review]
The purpose of this issue is to make sure `slcli vlan list` ALWAYS prints all vlans.

**Comment by edsonarios** [comment]
Screenshot of the test result
image

**Review by edsonarios — APPROVED** [review]
Apart from the comment, looks good to me

**Pull request merged**